src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -291,6 +350,9 @@ def main():
     output_dir = _make_output_dir(
         root_dir, loader_common_names.SOURCE_DATA_DIR, SOURCE, workspace_id
     )
+    collection_source_dir = _make_collection_source_dir(
+        root_dir, loader_common_names.SOURCE_DATA_DIR, collection, source_version
```

Comment:
If we want separate directories for "raw" data (`WS`, `NCBI`) and derived collections (`GTDB`, `PMI`) now is a good time to do it. `collections` makes the most sense to me, although that collides with the git repo, but maybe we could move that into a `git` directory.

@Tianhao-Gu what do you think?

Reply:
I think that will work. Given it's just softlink, we can probably use `collectionsdata/collection/load_ver/sourcedata`.

Reply:
So it's in the same directory as the batch directories?

Reply:
yea. together with the tools directories. `sourcedata`, `checkm2`, `microtrait`, etc.

Reply:
Oh, I forgot about the tool dirs. Ok, that seems reasonable

Reply:
Updated!

Reply:
:+1: